documentation/src/docs/rest/restapis.md

```diff
@@ -812,6 +812,43 @@ Fetch the properties of an existing reader group
 ```
 
 
+<a name="deletereadergroup"></a>
+### DELETE /scopes/{scopeName}/readergroups/{readerGroupName}
+
+#### Description
+Delete a reader group
+
+
+#### Parameters
+
+|Type|Name|Description|Schema|
+|---|---|---|---|
+|**Path**|**readerGroupName**  <br>*required*|Reader group name|string|
+|**Path**|**scopeName**  <br>*required*|Scope name|string|
+
+
+#### Responses
+
+|HTTP Code|Description|Schema|
+|---|---|---|
+|**204**|Successfully deleted the reader group|No Content|
+|**404**|Scope or reader group with given name not found|No Content|
```

Comment:
why **Scope or reader group**, It should be the reader group as this API is responsible for deleting RG not scope right?

Reply:
Done as suggested.